templates/project-ts/src/Add.test.ts

```diff
@@ -9,7 +11,7 @@ import { AddZkProgram } from './AddZkProgram';
  * See https://docs.minaprotocol.com/zkapps for more info.
  */
 
-const proofsEnabled = true;
+const proofsEnabled = false;
```

Comment:
Might want to revert this back.  I think `true` is the right default because it will catch all errors immediately.  Non-beginners can learn to turn this off, but beginners will want it on by default.

Reply:
I will revert proofsEnabled to true as a default. I mainly changed this to false for fast iterations when I was testing the tsconfig with the node test runner.